packages/mdc-toolbar/foundation.js

```diff
@@ -122,6 +121,7 @@ export default class MDCToolbarFoundation extends MDCFoundation {
     const newToolbarRowHeight = this.getRowHeight_();
     if (newToolbarRowHeight !== this.calculations_.toolbarRowHeight) {
       this.calculations_.toolbarRowHeight = newToolbarRowHeight;
+      this.initKeyRatio_();
```

Comment:
<img width="559" alt="screen shot 2017-06-13 at 15 00 06" src="https://user-images.githubusercontent.com/1894234/27099485-19fc8992-5049-11e7-895d-7ec4da30270e.png">

Here is the situation I mentioned. If you resize across the border (either from mobile to desktop or the other way around), the ratio will result in a incorrect state and the flexible space cannot be fully extended.